docs/mobile-apps/automated-testing/appium/appium-versions.md

```diff
@@ -0,0 +1,2769 @@
+---
+id: appium-versions
+title: Appium Versions
+sidebar_label: Appium Versions
+description: Appium Versions and EOL strategy.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::warning Appium 1 End-of-life
+The Appium core team does not maintain Appium 1.x anymore since the [1st of January 2022](https://github.com/appium/appium). This means that:
+
+- bugfixes will not be released for Appium 1.x anymore and require you to [migrate to Appium 2](./appium-2-migration.md)
+- Appium 1 and its packaged drivers are no longer 100% compatible with Android 13/iOS 16 and higher and require Appium 2 to run stably.
+
+Sauce Labs still supports Appium 1.x in limited versions, but we recommend [migrating to Appium 2](./appium-2-migration.md) to keep your tests up-to-date and compatible with the latest platform versions.
+:::
+
+## Selecting Appium Versions
+
+You can select a specific Appium version by using the `appiumVersion` capability as part of the `"sauce:options"`. The below examples show how to select the Appium version for your test and use Android with Chrome as an example. See our [Platform Configurator](https://saucelabs.com/products/platform-configurator#/) to help you construct your capabilities for the specific platform and Real Device/Android Emulator/iOS Simulator you want to test on.
```

Comment:
is the platform configurator default to Appium 2 already?

Reply:
yes, for EMUSIM it does, for RDC it's not defaulting. It's based on internal logic